pkg/diagnostics/v2/client.go

```diff
@@ -0,0 +1,43 @@
+package v2
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/dcos/dcos-diagnostics/api/rest"
```

Comment:
I'm not sure we should add this strict coupling here but rather only depend on HTTP and JSON as an API.